tasks/Alignment.wdl

```diff
@@ -91,6 +92,8 @@ task SamToFastqAndBwaMemAndMba {
         IS_BISULFITE_SEQUENCE=false \
         ALIGNED_READS_ONLY=false \
         CLIP_ADAPTERS=false \
+        ~{true='CLIP_OVERLAPPING_READS=true' false="" hard_clip_reads} \
+        ~{true='CLIP_OVERLAPPING_READS_OPERATOR=H' false="" hard_clip_reads} \
```

Comment:
The syntax here has changed, should be:
`~{true='CLIP_OVERLAPPING_READS=true' false="" hard_clip_reads} \
        HARD_CLIP_OVERLAPPING_READS=~{hard_clip_reads} \`

Reply:
Hi @jessicaway,

This is an older Pr, i'll be closing this out in favor of the 2.0 version of the workflow linked above.